frontend/src/components/object/ObjectPermission.vue

```diff
@@ -72,11 +75,35 @@ const updateObjectPermission = (value: boolean, userId: string, permCode: string
 
 onBeforeMount(() => {
   permissionStore.mapObjectToUserPermissions(props.objectId);
+  object.value = objectStore.findObjectById(props.objectId);
 });
 </script>
 
 <template>
   <div>
+    <div class="flex flex-row gap-6 pb-3">
+      <div>
+        <h3 class="pb-1">Public</h3>
+        <ul>
+          <li>This option toggles the file to be publicly available and accessible to anyone</li>
+          <li>To instead set explicit permissions, add users and use the options below</li>
+        </ul>
+      </div>
+      <div class="ml-4">
+        <ObjectPublicToggle
+          v-if="object && getUserId"
+          :bucket-id="object.bucketId"
+          :object-id="object.id"
+          :object-public="object.public"
+          :user-id="getUserId"
+        />
+      </div>
+    </div>
+
+    <div class="mt-1 mb-2">
+      <h3>User Permissions</h3>
+    </div>
```

Comment:
Might be able to get away with just an h3 with class attribute and not require the div wrapper?